### PR TITLE
Added provider AnimeTake

### PR DESCRIPTION
**Pull request by ArjixGamer**

### CODE REVIEW

**Comment by ArjixGamer** [comment]
Code quality might need some refining.
But it works fine.

File: anime_downloader/sites/animetake.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+
+class AnimeTakeEpisode(AnimeEpisode, sitename='animetake'):
+    def _get_sources(self):
+        soup = helpers.soupify(helpers.get(self.url).text)
+        episode_regex = r'gstoreplayer.source\s*=\s*({[\W\w]*?)player.on'
+        source = re.search(episode_regex, str(soup))
+
+        validate_json = [
+            ['type:', '"type":'], ['sources:', '"sources":'],
```
Comment (Blatzar) [inline anime_downloader/sites/animetake.py:43]: See https://github.com/vn-ki/anime-downloader/blob/master/anime_downloader/extractors/haloani.py#L44-L46 for a future proofed json fixer (only stuff like type: -> "type":)
Though you can use `[{|,][\s]*?[A-z]*?[^\"']:` as a better regex

File: anime_downloader/sites/animetake.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        return [
+            SearchResult(
+                title=x.find('center').text,
+                url='https://animetake.tv{}'.format(x.a['href']),
+                )
+            for x in soup.select('div.thumbnail')
+            ]
+
+    def _scrape_episodes(self):
+        soup = helpers.soupify(helpers.get(self.url).text)
```
Comment (Blatzar) [inline anime_downloader/sites/animetake.py:26]: No need for .text on something you soupify

File: anime_downloader/sites/animetake.py (hunk 1 above)
Comment (ArjixGamer) [reply anime_downloader/sites/animetake.py:43]: thx

**Pull request merged**